docs/import_process.md

```diff
@@ -0,0 +1,79 @@
+# Import process
+
+The import is started by the Django management command:
+
+```bash
+python manage.py import_xml path/to/xml/file.xml parent_page_id
+```
+
+but what happens afterwards is complicated, and highly configurable:
+
+<img src="import-flow-diagram.png">
```

Comment:
**Suggestion** I think the src is incorrect with the document now inside `docs` folder. It's probably better though to move the image into the same folder rather than update the src.

Reply:
Ah, good spot. Thanks.

Reply:
I have moved this.

Reply:
Thanks